README.md

```diff
@@ -10,342 +10,163 @@ Our goal is to provide the Magento community with comprehensive and quality tech
-
-### To minimize build time locally:
-
-1. Create a `_config.local.yml` file at the root of the project directory and exclude all versions except the one that you want to preview.
-
-   The following example will generate Magento 2.2 documentation only.
-
-   ```yaml
-    exclude:
-     - /community/
-     - /swagger/
-     - /vagrant/
-     - /guides/m1x/
-     - /guides/v2.0/
-     - /guides/v2.1/
-    # - /guides/v2.2/
-     - /guides/v2.3/
-
-    # Excluded in config.yml
-     - /scss/
-     - /bin/
-     - /node_modules/
-     - /vendor/
-     - /.*
-     - /Rakefile
-   ```
-
-1. Run the preview command:
-
-   ```bash
-   rake preview
-   ```
-
-   This command:
-
-   -  Checks your environment according to the dependencies in `Gemfile.lock`.
-   -  Removes the `_site/` directory, which contains previously generated preview files.
-   -  Generates a new preview and opens the landing page in a web browser.
-
-If you don't have the `_config.local.yml` file at the root of your `devdocs/` directory, the rake will generate all versions of the documentation.
-
-### To build v2.0 documentation
-
-Magento 2.0.18 was the final 2.0.x release. After March 2018, Magento 2.0.x stopped receiving security patches, quality fixes, and documentation updates.
-
-To ensure the quality of our public documentation for versions of Magento that are still supported and to avoid potential confusion about 2.0.x support, we removed all 2.0.x content from our public documentation websites on May 31, 2019.
+> Leave the serve terminal open and running. Every time you save changes to a file, it automatically regenerates the site so you can test the output immediately. Changing the `_config.yml` file or other YAML file with data or configuration requires a fresh build (stop and start the server again with `rake preview`).
 
-To build v2.0 documentation:
+### Exclude private repositories
 
-> Also, you can use the GitHub interface to read the source Markdown files instead of building the site locally. Navigate to the [`2.0`](https://github.com/magento/devdocs/tree/2.0) branch and click the file you want to view.
+If you do not have access to the private repositories required by Docfile, you can exclude them in `_config.local.yml` to avoid the link checking report about missing pages.
 
-1. Add an entry for 2.0 in the `Docfile.yml` file.
+Create a `_config.local.yml` file at the root of the project directory and exclude the paths you do not want to generate:
 
-   ```yaml
-   -
-     directory: guides/v2.0
-     repository: magento/devdocs
-     branch: 2.0
-     filter: false
-   ```
-
-1. Initialize the `2.0` branch in your local devdocs repository.
-
-   ```bash
-   rake init
-   ```
-
-1. Run the preview command.
+The following example will generate Magento 2.2 documentation only.
```

Comment:
```suggestion
The following example generates Magento 2.2 documentation only.
```

Reply:
This line should not be here. It's probably appeared after a merge conflict resolution.